cmd/bbolt/command_buckets.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	bolt "go.etcd.io/bbolt"
+)
+
+// newBucketsCommand creates a new command that prints a list of buckets in the given Bolt database.
+//
+// The path to a Bolt database must be specified as an argument.
+func newBucketsCommand() *cobra.Command {
+	var bucketsCmd = &cobra.Command{
+		Use:   "buckets <path>",
+		Short: "Print a list of buckets",
+		Long:  "Print a list of buckets in the given Bolt database\nThe path to a Bolt database must be specified as an argument",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return printBucketsList(cmd, args[0])
+		},
```

Comment:
Please try to keep consistent with other commands implementation.
```suggestion
		Short: "print a list of buckets",
		Args:  cobra.ExactArgs(1),
		RunE: func(cmd *cobra.Command, args []string) error {
			return bucketsFunc(args[0])
		},
```